src/App/Handler/StaticPageHandler.php

```diff
@@ -28,7 +28,55 @@ public function handle(ServerRequestInterface $request): ResponseInterface
 
         return new HtmlResponse($this->renderer->render(
             $this->normalize($routeResult->getMatchedRouteName()),
-            $request->getAttributes()
+            array_merge(
+                $request->getAttributes(),
+                [
+                    'commercialVendors' => [
```

Comment:
Shall I move this into the `\App\ConfigProvider` perhaps? Not sure best place for this configuration.

Reply:
Honestly, I would say:

- Create a unique handler for this, as it needs to pass data to the template.
- Put the data into either the config provider or a config file under `config/autoload/` (my preference, as I'd consider this _configuration_ of the page data).
- Have the factory for the handler pull the template renderer, a response factory, and the config, and pass them to the handler constructor so it can do its thing.